server/src/reactruntime.js

```diff
@@ -3,6 +3,8 @@ require("core-js");
 const React = require("react");
 const ReactDOM = require("react-dom");
 const linker = require("./linker");
+const { LocalizationProvider } = require("fluent-react/compat");
+const l10n = require("./l10n");
```

Comment:
`./linker` is a good example here – instead of that module actually finding the git revision, it has a `setGitRevision` function. There's two codepaths for injecting that content, one for the server and another for the client.

`reactrender.js` might be where the localization should get injected into the content. It could even be as simple as adding `window.localizedMessages = ${JSON.stringify(generateMessages(l10n.userLangs))` and then using that in `reactruntime`. Or maybe it would be better to add the messages to the model (which can happen in reactruntime), similar to gitRevision.